### DISCUSSION TITLE
How to Handle Discussions

### DISCUSSION THREAD

**Discussion by CliffAgius** [CommunityToolkit/Maui#1445] [Polls]
Hi All,

As mentioned on the last 2 monthly Community Toolkit stand-ups we have been Discussing how to handle the discussions here.

My suggestion is that we set a level of upvotes and if a discussion item passes this level we make a point of discussing on the next monthly stand-up.  From here we can then vote along with those in the chat on progressing this feature request or closing the discussion.

My feeling is that this will allow us as a community to openly discuss the points raised in the feature, if it gets voted as a yes in the stand-up we can then champion it and move it over to a feature on the project board for working on.  However if we vote No and we as a community feel it's not a good fit or a current available 3rd party OSS project covers it we close the discussion with a comment and link to the Stand-up video where it was discussed.

This will help to make sure that features listed are looked into and the community has a say in those that get added, I am suggesting that a starting level of 10 upvotes gets it discussed but we can amend the number.

Any feature that fails to meet the 10 vote threshold in say 6 months is closed automatically.

So please vote below if you feel this is a good idea, and feel free to comment below if you feel I am way off the mark with this idea after all this is a community decision.

**Comment by vhugogarcia** [comment]
I like this idea @CliffAgius , 

Here are my personal points:

- 10 up-votes may be a lot for the community toolkit. I think we should decrease it a little bit. I remember when I submitted an issue on Visual Studio Feedback site, I had to ask my friends and family to login and upvote it because otherwise nobody will hear me. I think the goal is that if a feature is requested at least have a minimum of 7 up-votes so it can be considered. Why 7? Well, 7 is 50% + 1 of the total of MCT core team, this means if nobody from outside voted for it, at least the MCT majority team voted for it. Also 7 is prime number.

- Clean up of the discussions. Right now there are around 50 discussions marked as "answered" but not closed. Also, there are maybe duplicated discussions, I will be happy to review the discussions, close all the answered, and try to identify the duplicated ones. Finally close those old discussions from 2021 that are not aplicable anymore.

Just my grain of salt. It is totally up to discussion and I agree with what all agree 😄

**Reply by CliffAgius** [reply]
I see the reason behind 7 and you had me at Prime number but the other argument is valid also.  I am happy to set it where the community agrees and my plan is to clean up the discussions based on this being voted for as you suggest removing old and closing those that are actioned already.

**Reply by vhugogarcia** [reply]
Awesome! Thanks @CliffAgius 👍🏻 😃

**Reply by ne0rrmatrix** [reply]
What about closed PR's where developer has either abandoned the PR or was unable to complete it? I started a PR that was championed and have been working on it in the background. I am hoping to reopen it in the future. 

But I don't see a process for what that involves or if it is even possible? I was working on Full Screen for MediaElement and have now made some further progress. I don't want to reopen it atm as I would like feedback on the method of fixing the root issue of original PR. The process for contacting a team member or getting an opinion of the solution I have is viable or if I should start over with it.

This is an example of the issues I face. Having a method for reopening old abandoned PR or resubmitting changes for review is either not obvious to a new developer(me) or has not been talked about. If you think I need to open a separate discussion I would be happy to do that.

Feel free to respond about opening closed PR's and/or starting development to further closed PR's where there is interest in the community for the feature. Another note it would be nice to get feedback on the process of checking an idea to see if someone is on right track. I would appreciate any feedback about that.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
